dbms/src/Common/MyTime.cpp

```diff
@@ -58,12 +59,17 @@ int32_t adjustYear(int32_t year)
 
 void scanTimeArgs(const std::vector<String> & seps, std::initializer_list<int *> && list)
 {
+    auto log = &Poco::Logger::get("LRUCache");
+    LOG_FMT_INFO(log, "seps.size: {}, list.size: {}", seps.size(), list.size());
```

Comment:
seems we can replace `LOG_FMT_XXX` with `LOG_XXX` now. refer to #5511

Reply:
> seems we can replace `LOG_FMT_XXX` with `LOG_XXX` now. refer to #5511

Some of the logs should be removed.